docs/articles/documentation/using-testcafe/common-concepts/using-browsers.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+title: Using Browsers
+permalink: /documentation/using-testcafe/common-concepts/using-browsers.html
+---
+# Using Browsers
+
+TestCafe allows you to easily set up testing in different web browsers: a local, remote, or custom browser supplied by a [browser provider plugin](#browser-provider-plugins).
```

Comment:
`custom browser` means a web browser application tailored to the needs of a specific person... which is not what we want to say. we mean an arbitrary web browser - just any browser.

and the browser provider does not `supply` browsers.

and neither `custom` nor `arbitrary` stand together with `local` and `remote` because neither is the third alternative to `local` and `remote`